apps/explorer/app/block/[id]/page.tsx

```diff
@@ -34,21 +34,30 @@ export const revalidate = 0
 
 export default async function Page({ params }) {
   const id = params?.id as string
-  const [b, error] = await to(
-    siaCentral.block({
-      params: {
-        id,
-      },
-    })
+
+  // 1. Grab the tip at this height
+  const [tipInfo, tipInfoError] = await to(
+    explored.consensusTipByHeight({ params: { height: Number(id) } })
   )
 
-  if (error) {
-    throw error
+  if (tipInfoError) {
+    throw tipInfoError
+  }
+  if (!tipInfo) {
+    throw notFound()
   }
 
-  if (!b?.block) {
+  // 2. Get the block using the id from the previous request
+  const [block, blockError] = await to(
```

Comment:
Do we plan to add a height-based endpoint so we don't have to sequence 2 requests?

Reply:
As far as I understand, we do not currently plan to add that endpoint. @n8maninger mentioned this to me when we were talking through a process question.

Reply:
My opinion is `/block/[id]` should always be the block hash. Right now it looks like SiaScan supports both, but that's not consistent with any other blockchain element. If we add support for orphans in the future, there may be two blocks with the same height.

Reply:
👍🏼